app/views/collection_drafts/show.html.erb

```diff
@@ -22,15 +22,33 @@
     <section class="action wide-content">
       <div class="wide-content-inside">
         <% if resource_name == 'collection_draft' %>
-          <% if get_resource.new_record? || (@errors && @errors.size > 0) %>
+          <% if get_resource.new_record? || (!@is_revision && @errors && @errors.size > 0) %>
```

Comment:
Consider rewriting to:
if (get_resource.new_record? || (@errors && @errors.size > 0)
  if (@is_revision)
       do this
  else 
       do that
end

Reply:
yeah that seems cleaner, made the change